DuckDuckGo.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -50,8 +50,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/duckduckgo/duckduckgo-autofill.git",
       "state" : {
-        "revision" : "3a9606fd26e9a54bf369cc241e6fa3b2571eb13a",
-        "version" : "16.2.1"
+        "revision" : "b5976277a68e18f93edb0d853889e461e06557fa",
```

Comment:
This is being pushed on me automatically so I'd just submit it.

Reply:
Yeah this is fine, I think this is a small regression with the monorepo where the release automation is only updating BSK and not the Package.resolved values of the projects.